### DISCUSSION TITLE
Installer won't allow static IP

### DISCUSSION THREAD

**Discussion by MatrixInfosec** [Security-Onion-Solutions/securityonion#7901] [Unsupported Versions]
I'm setting up SecOnion in a VM on XCp-NG. The installer won't allow me to set a static IP for the management interface. When I try to set one, it acts like it is accepting the settings until I get the summary screen, The summary screen shows a DHCP address that I didn't set. 

Any idea how to fix this?

**Comment by dougburks** [comment]
Are you using our Security Onion ISO image or are you performing a manual installation?

If manual, are you installing on CentOS 7, Ubuntu 18.04, or Ubuntu 20.04?

Are you able to share screenshots?

Have you tried outside of XCP-ng to see if you get anything different?

**Reply by MatrixInfosec** [reply]
I am using the iso. This morning, I tried to configure the network again, using the setup command. This time it said it successfully reconfigured, however when I run ifconfig -a. it still shows the DHCP address 192.168.2.231. Eth1 is the interface.
![seconion](https://user-images.githubusercontent.com/62116817/167184886-e2ec77cf-6f88-4087-803e-4382f65d3b06.png)

**Reply by dougburks** [reply]
Have you tried outside of XCP-ng to see if you get anything different?
